src/composables/ViewRoutingSlip/usePaymentInformation.ts

```diff
@@ -51,6 +70,20 @@ export default function usePaymentInformation (_, context) {
     return linkedRoutingSlips.value && linkedRoutingSlips.value.children.length > 0 && linkedRoutingSlips.value.children[0].totalUsd && linkedRoutingSlips.value.children[0].totalUsd > 0
   })
 
+  function adjustRoutingSlipHandler () {
+    try {
+      adjustRoutingSlip()
+      isEditable.value = !isEditable.value
```

Comment:
Similar to adjustRoutingSlipStatus? why not just call it?

Reply:
thanks for catching up! changed to function call